configuration.rst

```diff
@@ -176,12 +177,33 @@ server-unix-socket-mode
 
     server-unix-socket-mode = "660"
 
+.. _log-level:
+
+log-level
+---------
+
+  Specifies the level of information to be logged while running PostgREST.
+
+  .. code:: bash
+
+      # No requests are logged, only startup and db connection recovery messages are.
+      log-level = "crit"
+
+      # All the "crit" level events plus server errors (status >= 500) are logged
```

Comment:
I'm not sure if the `>=` usage is confusing. Maybe, it should be replaced for the status class?

```suggestion
      # All the "crit" level events plus server errors (status 5xx) are logged
```